aten/src/ATen/native/ReduceOps.cpp

```diff
@@ -435,7 +435,7 @@ static Tensor& norm_out(Tensor &result, const Tensor &self, optional<Scalar> opt
   TORCH_CHECK(self.device().type() == DeviceType::CPU || self.device().type() == DeviceType::CUDA,
               "norm only supports CPU AND CUDA device type, got: ", self.device().type());
   TORCH_CHECK(self.layout() == Layout::Strided,
-              "norm only supports strided layout, got: ", self.layout());
+              "norm_out only supports strided layout, got: ", self.layout());
```

Comment:
This is a user-facing error but `norm_out` is an internal name. The prior check also says `norm.`